corehq/apps/reports/views.py

```diff
@@ -276,10 +278,33 @@ def export_data_async(request, domain):
     except ValueError:
         return HttpResponseBadRequest()
     assert(export_tag[0] == domain)
+    format = request.GET.get("format", Format.XLS_2007)
+    filename = request.GET.get("filename", None)
+    previous_export_id = request.GET.get("previous_export", None)
 
     filter = util.create_export_filter(request, domain, export_type=export_type)
```

Comment:
Yeah, `format` and `filter` shadow the built-ins—unfortunately this is the pattern all over couchexport so just sticking with that convention for now